script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -929,7 +929,7 @@ A final paradise, sprawled among the mountains.
 [sha:62e43bd98536e70dc341b01ba02c5fe8e4b2b461]{
 -- Page 99, Offset 29779.
 -- 星の内海を思わせる黎明の原野。
-A dawn wilderness that calls to mind a cove of stars.
+A dawn wilderness that calls to mind the %{i}Inner-sea of the Planet%{/i}.
```

Comment:
don't need hyphen

Reply:
We could go with "Inner Sea" (though personally I think it looks better with the hyphen, hence why I put it as such in the spreadsheet). 

I suppose another possibility is just to be like "lol, time to make "Innersea" into a world by ourselves", but that requires more deliberate conversation.

Reply:
I'd also vote for no hyphen